src/main/kotlin/codecov/Request.kt

```diff
@@ -1,40 +1,31 @@
 // Import to make requests
 package codecov
 
-import java.util.*
 import kotlin.system.exitProcess
-import com.beust.klaxon.Parser
-import com.beust.klaxon.JsonObject
-
 
 object Request {
+
     @JvmStatic
     fun main(args: Array<String>) {
         val url = "https://codecov.io/api/gh/codecov/Kotlin-Standard"
-        val token = System.getenv("API_KEY")
-        // Sleep 1 minute    
+        // Sleep 1 minute
         Thread.sleep(60_000)
         // Make request
-        val response = khttp.get(
-            url = url,
-            params = mapOf("token" to token)
-        )
+        val response = khttp.get(url = url)
```

Comment:
just curious @pavlospt, what's the reason for removing `token`?

Reply:
It works without it just fine :)

Reply:
@pavlopst, I believe we have implemented tokenless uploading for some of the major CI/CD providers (e.g. TravisCI, GitHub Actions, CircleCI) on public repositories, but the `token` is still required for private repositories and non-tokenless CI/CDs. 

I am not confident that this can be taken out, but if you have a build showing different, please let me know. Pending reverting the `token` (or proving me wrong), I'll merge this in.

Reply:
@thomasrockhu my local builds and try-outs certainly did not include a token and was successful runs. Maybe it is needed for private repos but is that the point of this sample?

Reply:
@pavlospt, my mistake, this is a different `token` than what I was expecting after further review. Thanks for the work here!